tests/_support/AcceptanceTester.php

```diff
@@ -155,6 +155,40 @@ public function selectOptionInReactSelect($value, $selector) {
     $i->pressKey($selector . ' input', WebDriverKeys::RETURN_KEY);
   }
 
+  /**
+   * Wrapper around fillField, emit change event
+   *
+   * useful for emiting change events in react input components
+   *
+   * @param string $element - selectable by document.querySelector
+   * @param string $value
+   */
+  public function fillFieldWithOnChangeEvent(string $element, string $value = '') {
```

Comment:
Kindly check this Stackoverflow page for more information on why this is required: https://stackoverflow.com/questions/39065010/why-react-event-handler-is-not-called-on-dispatchevent